tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -94,9 +95,11 @@ Get RHODS version
     #@{list} =  OpenShiftCLI.Get  kind=ClusterServiceVersion  label_selector=olm.copiedFrom=redhat-ods-operator
     #&{dict} =  Set Variable  ${list}[0]
     #Log  ${dict.spec.version}
-    ${ver} =  Run  oc get csv -n redhat-ods-operator | grep "rhods-operator" | awk '{print $1}' | sed 's/rhods-operator.//'
-    Log  ${ver}
-    [Return]  ${ver}
+    IF  "${RHODS_VERSION}" == "${None}"
+         ${RHODS_VERSION} =  Run  oc get csv -n redhat-ods-operator | grep "rhods-operator" | awk '{print $1}' | sed 's/rhods-operator.//'
```

Comment:
does it replace the global variable or create a local copy?

Reply:
it creates local copy

Reply:
Maybe an argument  set-global-variable=True (default True) could be added (instead of argument force-reload, proposed in another comment)

Reply:
hm since you this kw will run in the RHOSi Setup, maybe here we don't need to return any value but just replace the Global variable value. What do you think?

Reply:
I have used force fetch as an optional parameter

Reply:
@bdattoma I think if i remove return if anyone is using the keyword directly for some TC they need modify the suite I don't want TC to failed if they are using it

Reply:
yes the TCs need to ne updated. The refactoring is becoming always bigger xD but okay